src/Misc/layoutbin/hashFiles/index.js

```diff
@@ -1057,7 +1057,9 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
     });
 };
 Object.defineProperty(exports, "__esModule", { value: true });
+exports.hashFiles = exports.create = void 0;
 const internal_globber_1 = __webpack_require__(297);
```

Comment:
I think modifying this file is causing the test to fail for some reason, can we put in a fix for that test as a part of this pr?